wgpu/gui/qt.py

```diff
@@ -244,12 +247,27 @@ def _key_event(self, event_type, event):
         }
         self._handle_event_and_flush(ev)
 
+    def _char_input_event(self, char, char_str):
+        ev = {
+            "event_type": "char",
+            "char": char,
+            "char_str": char_str,
```

Comment:
Only `char_str` is used, and `char` is not always set, so cannot be relied on. So let's use just one field.